admin/client/src/components/PopoverTooltip/PopoverTooltip.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jQuery';
+import 'tetherWrapper';
+import 'bootstrap-popover';
+import 'bootstrap-tooltip';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class PopoverTooltip extends React.Component {
+  componentDidMount() {
+    const el = ReactDOM.findDOMNode(this);
+
+    // Enable BS Popovers and tooltips
+    $(el).find('[data-toggle="tooltip"]').tooltip();
+    $(el).find('[data-toggle="popover"]').popover({ html: true });
+
+    // Accessibility support for popover focus (requires anchor element)
+    $(el).find('.popover-dismiss').popover({ trigger: 'focus' });
+  }
+
+  // TODO allow for custom popover/tooltip trigger
+  // eg. (classes, contents, data-toggle, placement, icon and text)
+  // Add the possibility to use FormAction component as a trigger
```

Comment:
Yeah I agree, I already mention earlier that a story has been created to fix this but it isn't really part of our current work, so I was just trying to clean up the code that that is relevant. I don't think I'm the right person to be doing the React stuff though.